docker-compose.yml

```diff
@@ -1,52 +1,65 @@
-version: '3.4'
+version: '2.4'
+
+x-app-service-template: &app
+  image: anycable-demo:0.6.0
+  build:
+    context: .
+    dockerfile: .dockerdev/Dockerfile
+    args:
+      RUBY_VERSION: "2.5.7"
+      NODE_MAJOR: '12'
+      BUNDLER_VERSION: '1.17.1'
+      HIVEMIND_VERSION: '1.0.6'
```

Comment:
Why do we need hivemind in Docker 😃?

Reply:
Ok, I found it: we use it for specs.
And, actually, we can drop it, because `Profile.spec` contains only a single command.

Reply:
Here is also `Procfile.spec_erl` with two commands. Do we need it?

Reply:
Nope. ErlyCable support is suspended, so we can drop everything related to it.